<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkWatershedToolGUIControls</class>
 <widget class="QWidget" name="QmitkWatershedToolGUIControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>352</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>100000</width>
    <height>100000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkOtsuToolWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="advancedSettingsLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="levelLabel">
       <property name="text">
        <string>Level:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="thresholdLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Threshold:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="ctkSliderWidget" name="thresholdSlider" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="ctkSliderWidget" name="levelSlider" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QmitkSimpleLabelSetListWidget" name="m_selectionListWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>10000000</width>
       <height>10000000</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="previewButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>100000</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_ConfSegButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>100000</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Confirm Segmentation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_CheckProcessAll">
     <property name="toolTip">
      <string>Process/overwrite all time steps of the dynamic segmentation and not just the currently visible time step.</string>
     </property>
     <property name="text">
      <string>Process all time steps</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_CheckCreateNew">
     <property name="toolTip">
      <string>Add the confirmed segmentation as a new segmentation instead of overwriting the currently selected.</string>
     </property>
     <property name="text">
      <string>Create as new segmentation</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkSimpleLabelSetListWidget</class>
   <extends>QWidget</extends>
   <header>QmitkSimpleLabelSetListWidget.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSliderWidget</class>
   <extends>QWidget</extends>
   <header location="global">ctkSliderWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
